src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -234,14 +289,18 @@ private void initTasks()
         this.tasks.addTask(++priority, new EntityAISleep(this));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)